generators/chipyard/src/main/scala/Subsystem.scala

```diff
@@ -52,25 +52,37 @@ trait HasChipyardTiles extends HasTiles
   // TODO: investigate why
   val tiles = allTilesInfo.sortWith(_._1.hartId < _._1.hartId).map {
     case (param, crossing) => {
-      val (tile, rocketLogicalTree) = param match {
+
+      val tile = param match {
         case r: RocketTileParams => {
-          val t = LazyModule(new RocketTile(r, crossing, PriorityMuxHartIdFromSeq(rocketTileParams), logicalTreeNode))
-          (t, t.rocketLogicalTree)
+          LazyModule(new RocketTile(r, crossing, PriorityMuxHartIdFromSeq(rocketTileParams), logicalTreeNode))
         }
         case b: BoomTileParams => {
-          val t = LazyModule(new BoomTile(b, crossing, PriorityMuxHartIdFromSeq(boomTileParams), logicalTreeNode))
-          (t, t.rocketLogicalTree) // TODO FIX rocketLogicalTree is not a member of the superclass, both child classes define it separately
+          LazyModule(new BoomTile(b, crossing, PriorityMuxHartIdFromSeq(boomTileParams), logicalTreeNode))
         }
         case a: ArianeTileParams => {
-          val t = LazyModule(new ArianeTile(a, crossing, PriorityMuxHartIdFromSeq(arianeTileParams), logicalTreeNode))
-          (t, t.rocketLogicalTree) // TODO FIX rocketLogicalTree is not a member of the superclass, both child classes define it separately
+          LazyModule(new ArianeTile(a, crossing, PriorityMuxHartIdFromSeq(arianeTileParams), logicalTreeNode))
         }
       }
+      //val (tile, rocketLogicalTree) = param match {
+      //  case r: RocketTileParams => {
+      //    val t = LazyModule(new RocketTile(r, crossing, PriorityMuxHartIdFromSeq(rocketTileParams), logicalTreeNode))
+      //    (t, t.rocketLogicalTree)
+      //  }
+      //  case b: BoomTileParams => {
+      //    val t = LazyModule(new BoomTile(b, crossing, PriorityMuxHartIdFromSeq(boomTileParams), logicalTreeNode))
+      //    (t, t.rocketLogicalTree) // TODO FIX rocketLogicalTree is not a member of the superclass, both child classes define it separately
+      //  }
+      //  case a: ArianeTileParams => {
+      //    val t = LazyModule(new ArianeTile(a, crossing, PriorityMuxHartIdFromSeq(arianeTileParams), logicalTreeNode))
+      //    (t, t.rocketLogicalTree) // TODO FIX rocketLogicalTree is not a member of the superclass, both child classes define it separately
+      //  }
+      //}
       connectMasterPortsToSBus(tile, crossing)
       connectSlavePortsToCBus(tile, crossing)
 
-      def treeNode: RocketTileLogicalTreeNode = new RocketTileLogicalTreeNode(rocketLogicalTree.getOMInterruptTargets)
-      LogicalModuleTree.add(logicalTreeNode, rocketLogicalTree)
+      //def treeNode: RocketTileLogicalTreeNode = new RocketTileLogicalTreeNode(rocketLogicalTree.getOMInterruptTargets)
```

Comment:
Why is this deleted?

Reply:
We don't use the this OM so i think we should remove our support for it.

Reply:
My impression when I last looked at it a while ago was that it was tied to the DTS. I'm not opposed to getting rid of support for it as long as it doesn't affect anything else.

Reply:
ill check that the generated DTS is the same

Reply:
@abejgonzalez The DTS seems unaffected by this change

Reply:
They're (unfortunately) parallel implementations that share some common infrastructure. If anything, OM builds/relies on the `Device` snippets used by DTS, not vice versa.

Reply:
Thanks for the clarification